Makefile

```diff
@@ -6,14 +6,14 @@ OPERATOR_ADDRESS := 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266
 BOOTSTRAP_OUTPUT_PATH := $(shell pwd)/build/bootstrap.json
 BOOTSTRAPPER_COMMIT := b0b647500c2ae3e3b0d99e345fa652989bca4726
 BOOTSTRAPPER_PATH := $(shell pwd)/madara-bootstrapper
-VERIFIER_ADDRESS := 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
+VERIFIER_ADDRESS := 0xa513E6E4b8f2a923D98304ec87F64353C4D5C853
```

Comment:
this verifier address is different. When we deploy using the cargo test method it is 0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512.